draft-ietf-tls-tls13.md

```diff
@@ -3128,6 +3168,10 @@ a KeyUpdate with the old key is received before accepting any messages
 encrypted with the new key. Failure to do so may allow message truncation
 attacks.
 
+Implementations that receive a KeyUpdate without having advertised
+key_update in the post_handshake extension MUST terminate the
```

Comment:
This seems to allow the scenario where a client advertises KeyUpdate support, a server does not include KeyUpdate in its EncryptedExtensions, but then the server goes and sends a KeyUpdate anyway.
That's a pretty silly scenario, but is an asymmetry that could be smoothed over.